PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -527,4 +534,24 @@ function appendRuleDocUrls(sarifReport: string) {
         })
     })
     return JSON.stringify(sarifJson);
+}
+
+function verifyDTPService() {
+    let url = dtpBaseUrl + "grs/api/v1/dtpServices";
+    return axios.default.get(url, {
+        httpsAgent: httpsAgent,
+        auth: {
+            username: dtpUsername,
+            password: dtpPassword
+        }
+    }).then(() => {
+        isDTPServiceAvailable = true;
+    }).catch((error) => {
+        let status = error.response ? error.response.status : -1;
+        if (status === 401) {
```

Comment:
We don't need to check for 401 specifically.
Please make the error message consistent with https://github.com/parasoft/parasoft-findings-vsts/pull/27/files